b2share/config.py

```diff
@@ -47,6 +41,180 @@
     account_json_loader, account_json_patch_loader,
 )
 
+from invenio_search.config import SEARCH_ELASTIC_HOSTS
+
+SEARCH_ELASTIC_HOSTS = []
+
+for elastic_host in os.environ.get("ELASTIC_HOSTS", "localhost:9200").split(','):
+    try:
+        (host, port) = elastic_host.split(':')
+    except:
+        host = elastic_host
+        port = 9200
+
+    SEARCH_ELASTIC_HOSTS.append(dict(host=host, port=port, use_ssl=False, ssl_show_warn=False))
+
+def _(x):
+    """Identity function used to trigger string extraction."""
+    return x
+
+# Rate limiting
+# =============
+#: Storage for ratelimiter.
+RATELIMIT_STORAGE_URL = os.environ.get("INVENIO_RATELIMIT_STORAGE_URL", 'redis://localhost:6379/3')
+
+# I18N
+# ====
+#: Default language
+BABEL_DEFAULT_LANGUAGE = 'en'
+#: Default time zone
+BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
+#: Other supported languages (do not include the default language in list).
+I18N_LANGUAGES = [
+    # ('fr', _('French'))
+]
+
+# Base templates
+# ==============
+#: Global base template.
+BASE_TEMPLATE = 'b2share_main/base.html'
+#: Cover page base template (used for e.g. login/sign-up).
+COVER_TEMPLATE = 'invenio_theme/page_cover.html'
+#: Footer base template.
+MAIN_TEMPLATE = 'invenio_theme/footer.html'
+#: Header base template.
+HEADER_TEMPLATE = 'invenio_theme/header.html'
+#: Settings base template.
+SETTINGS_TEMPLATE = 'invenio_theme/page_settings.html'
+
+# Theme configuration
+# ===================
+#: Site name
+THEME_SITENAME = _('B2SHARE')
+#: Use default frontpage.
+THEME_FRONTPAGE = True
+#: Frontpage title.
+THEME_FRONTPAGE_TITLE = _('B2SHARE')
+#: Frontpage template.
+THEME_FRONTPAGE_TEMPLATE = 'b2share_main/page.html'
+
+# Email configuration
+# ===================
+#: Email address for support.
+SUPPORT_EMAIL = "info@eudat.eu"
+#: Disable email sending by default.
+MAIL_SUPPRESS_SEND = True
+
+# Assets
+# ======
+#: Static files collection method (defaults to copying files).
+COLLECT_STORAGE = 'flask_collect.storage.file'
+
+# Accounts
+# ========
+#: Email address used as sender of account registration emails.
+SECURITY_EMAIL_SENDER = SUPPORT_EMAIL
+#: Email subject for account registration emails.
+SECURITY_EMAIL_SUBJECT_REGISTER = _(
+    "Welcome to B2SHARE!")
+#: Redis session storage URL.
+CACHE_REDIS_URL=os.environ.get("INVENIO_CACHE_REDIS_URL", 'redis://localhost:6379/0')
+ACCOUNTS_SESSION_REDIS_URL = os.environ.get("INVENIO_ACCOUNTS_SESSION_REDIS_URL", 'redis://localhost:6379/1')
+#: Enable session/user id request tracing. This feature will add X-Session-ID
+#: and X-User-ID headers to HTTP response. You MUST ensure that NGINX (or other
+#: proxies) removes these headers again before sending the response to the
+#: client. Set to False, in case of doubt.
+ACCOUNTS_USERINFO_HEADERS = True
```

Comment:
Note that V3 uses invenio-cache: https://github.com/inveniosoftware/invenio-cache

It might be that V2 codebase invenio modules use Redis directly and now they handle their caching needs through invenio-cache.